java/de/yessoft/android/mohammad/model/NewsItem.java

```diff
@@ -0,0 +1,87 @@
+package de.yessoft.android.mohammad.model;
+
+import android.util.ArrayMap;
+import android.util.Log;
+
+import com.google.firebase.firestore.DocumentSnapshot;
+
+import java.util.List;
+import java.util.Map;
+
+public class NewsItem {
+    private static final String TAG = "ML_NewsItem";
+
+    private static final String KEY_HEADING = "heading";
+    private static final String KEY_IMG = "img";
+    private static final String KEY_SUMMERY = "summery";
+    private static final String KEY_FULL_NEWS_ID = "full_news_id";
+
+    private String Heading;
+    private String Summery;
+    private String Img;
+    private String FullNewsId;
+
+    // region Construct
+    private NewsItem() {
+    }
+
+    public NewsItem(DocumentSnapshot documentSnapshot) {
```

Comment:
this constructor should be removed , and create new mapper class 
the mapper job is to convert from : DocumentSnapshot documentSnapshot to NewsItem

and we should do some validation before mapping

Reply:
OK